features/0023-did-exchange/README.md

```diff
@@ -286,13 +148,15 @@ The _invitee_ will provision a new DID according to the DID method spec. For a P
 
 An invitation is presented in one of two forms:
 
-* An explicit invitation with its own `@id` as seen in the three examples [above](#invitation-message-with-public-invitation-did).
-* An [implicit](#implicit-invitation) invitation contained in a DID document's [`service`](https://w3c-ccg.github.io/did-spec/#service-endpoints) attribute.
+* An explicit out-of-band invitation or request with its own `@id`.
+* An implicit invitation contained in a DID document's [`service`](https://w3c-ccg.github.io/did-spec/#service-endpoints) attribute that conforms to the [DIDComm conventions](../0067-didcomm-diddoc-conventions/README.md#service-conventions).
 
 When a `request` responds to an explicit invitation, its `~thread.pthid` MUST be equal to the `@id` property of the invitation.
 
 When a `request` responds to an implicit invitation, its `~thread.pthid` MUST contain a [DID URL](https://w3c-ccg.github.io/did-spec/#dfn-did-url) that resolves to the specific `service` on a DID document that contains the invitation.
 
+_Note: The `request` message referred to above is the request message of the protocol used such as Connections or DID Exchange, NOT an Out-of-Band `request` message._
```

Comment:
This can be removed now that there is no `request` OOB message.

Reply:
+1